display_list/dl_color_unittests.cc

```diff
@@ -19,6 +19,14 @@ static void arraysEqual(const uint32_t* ints,
   }
 }
 
+TEST(DisplayListColor, DefaultValue) {
+  DlColor color;
+  EXPECT_EQ(color.getAlphaF(), 1.f);
+  EXPECT_EQ(color.getRedF(), 0.f);
+  EXPECT_EQ(color.getGreenF(), 0.f);
+  EXPECT_EQ(color.getBlueF(), 0.f);
```

Comment:
Perhaps check the default ColorSpace as well.

Reply:
Done.